server/src/main/java/io/crate/execution/dml/BulkShardResponseListener.java

```diff
@@ -22,6 +22,7 @@
 package io.crate.execution.dml;
 
 import java.util.concurrent.CompletableFuture;
+import java.util.function.BiConsumer;
```

Comment:
Please revert all changes inside this file, they are not needed. Set the `continueOnError` at `UpdateById` based on `plannerContext.transactionContext().sessionSettings().allowFailOnPartialWrites()` instead.